File: LinkedList/LinkedList-Introduction/intro.py
class Node:
    def __init__(self, data, next=None):
        self.data = data
        self.next = next


arr = [2, 5, 8, 7]

# Create first node
y = Node(arr[0])

print(y)        # prints object reference (like memory address)
print(y.data)   # prints 2

File: LinkedList/LinkedList-Introduction/intro.js
class Node {
  constructor(data, next = null) {
    this.data = data;
    this.next = next;
  }
}

const arr = [2, 5, 8, 7];

// Create first node
const y = new Node(arr[0]);

console.log(y);        // prints Node { data: 2, next: null }
console.log(y.data);   // prints 2
